menu/cakes/cakeA.html

```diff
@@ -0,0 +1,409 @@
+                </label>
+            
+            
+              </div>
+            
+              <label>
+                Additional Info:
+                <textarea name="additional info"></textarea>
+              </label>
+              <!-- your other form fields go here -->
+              <button type="submit" class="button big">Order Now</button>
+            </form>
+          </div>
+        </section>
+
+        <!-- Section -->
+        <!-- <section>
+									<header class="major">
+										<h2>Erat lacinia</h2>
+									</header>
+									<div class="features">
+										<article>
+											<span class="icon fa-gem"></span>
+											<div class="content">
+												<h3>Portitor ullamcorper</h3>
+												<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
+											</div>
+										</article>
+										<article>
+											<span class="icon solid fa-paper-plane"></span>
+											<div class="content">
+												<h3>Sapien veroeros</h3>
+												<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
+											</div>
+										</article>
+										<article>
+											<span class="icon solid fa-rocket"></span>
+											<div class="content">
+												<h3>Quam lorem ipsum</h3>
+												<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
+											</div>
+										</article>
+										<article>
+											<span class="icon solid fa-signal"></span>
+											<div class="content">
+												<h3>Sed magna finibus</h3>
+												<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
+											</div>
+										</article>
+									</div>
+								</section> -->
+
+        <!-- RELATED PRODUCTS -->
+        <!-- TODO - fix links to related products -->
+        <!-- Section -->
+        <section>
+          <header class="major">
+            <h2>Related Products</h2>
+          </header>
+          <div class="posts related-posts">
+            <article>
+              <a href="#" class="image"><img src="../../images/cakes/cake_pic04.jpg"
+                  alt="kid's rodeo themed birthday cake" /></a>
+              <h3>Cake B</h3>
+              <p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante
+                interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
+              <ul class="actions">
+                <li><a href="../../orders.html" class="button prefill-button" data-order-type="birthday-kids">Order This</a>
```

Comment:
I think this is your plan already but just as a note for posterity: in the future we will change this to cakeB.html since orders.html is deprecated.